public/components/security/users/users.tsx

```diff
@@ -118,10 +118,16 @@ export const Users = () => {
           </EuiTitle>
         </EuiPageContentHeaderSection>
         <EuiPageContentHeaderSection>
-          <div>
-            <EuiButton onClick={() => setIsCreateFlyoutVisible(true)}>Create user</EuiButton>
-            {createFlyout}
-          </div>
+          {
+            !rolesLoading
```

Comment:
you could use `&&` instead of a nullable ternary operator 
```
{ 
  !rolesLoading && <div> .. </div>
}
```

Reply:
All nullable ternary operators have been removed.